lineage/import_lineage.py

```diff
@@ -199,17 +189,17 @@ def MCF10A(condition: str):
 
     elif condition == "EGF":
         data1 = import_MCF10A("lineage/data/MCF10A/EGF_1.csv")
-        # data2 = import_MCF10A("lineage/data/MCF10A/EGF_2.csv")
```

Comment:
Is it possible to import these files but skip the lineages with bad data?

Reply:
Yes, it is done in this pr.